content/docs/reference/reproducibility.md

```diff
@@ -0,0 +1,41 @@
++++
+title="Reproducible Builds"
+weight=9
++++
+
+## Summary
+The Cloud Native Buildpacks project aims to create "Reproducible Builds" of container images. For image creation commands (`create-builder`, `package-buildpackag`, `build`) `pack` aims to create in a reproducible fashion. "Reproducible" is hard to define but we'll do so by example:
+
+---
+Running `pack build` produce a container image with the same image ID (*local* case)
+
+**Given**:
+- A workspace directory containing the same source code
+- A builder image with a *given*  
+- One or more buildpacks that produce identical layers given their input*
+
+---
+Running `pack build --publish` produce a container image with the same image digest (*remote* case)
+
+**Given**:
+- A workspace directory containing the same source code
+- A builder image with a *given*  
+- One or more buildpacks that produce identical layers given their input
+
+### Consequences and Caveats
+
+We achieve reproducible builds by "zeroing" various timestamps of the layers that `pack` creates. When images are inspected (via something like `docker inspect`) they may have confusing creation times:
```

Comment:
"layers that `pack` creates" -- is all of the zeroing done by pack? I thought the exporter was responsible for some of this. I think it's worth being precise about this.

Reply:
I agree that being precise here is useful, but the fact that pack creates these layers via the lifecycle is getting a bit into the weeds for this is as a first pass.  I was aiming for a higher level overview of the behavior that users will observe. That said, I the goal in getting this out is to point folks who have questions about reproducibility here and then further improve the documentation. I could very well image having a more technical/architectural explanation of this concept on this page at some later point, but I don't know if we need that _right_ now. 

All that said, I'm open to being wrong on that!

Reply:
Sounds good, we can always add more detail later :)